libs/dh/core/shell/src/lib/dh-primary-navigation.component.html

```diff
@@ -53,11 +53,21 @@
       >{{ transloco("imbalancePrices") }}</watt-nav-list-item
     >
 
-    <ng-container *dhFeatureFlag="'reports-v2'">
-      <watt-nav-list-item *dhPermissionRequired="['fas']" [link]="getLink('reports')">{{
-        transloco("reports")
-      }}</watt-nav-list-item>
-    </ng-container>
+    <watt-nav-list [expandable]="true" [title]="transloco('reports.menuGroupLabel')">
+      <ng-container *dhFeatureFlag="'reports-v2'">
+        <watt-nav-list-item *dhPermissionRequired="['fas']" [link]="getLink('reports')">{{
+          transloco("reports.overview")
+        }}</watt-nav-list-item>
+      </ng-container>
+      <ng-container *dhFeatureFlag="'missing-measurements-log'">
+        <watt-nav-list-item
+          *dhPermissionRequired="['missing-measurements-log:view']"
+          [link]="getWholesaleLink('missing-measurements-log')"
+        >
+          {{ transloco("reports.missingMeasurementsLog") }}
+        </watt-nav-list-item>
+      </ng-container>
+    </watt-nav-list>
```

Comment:
suggestion: as mentioned in our chat, let's keep the menu item as it was and instead redirect to the "right" page/tab based on the user's permissions.

Reply:
Missing measurements should not be a tab, see sitemap: https://www.figma.com/board/0J16vedi30UohMabwCi1FU/Sitemap-DH3?node-id=0-1&p=f&t=hGGIKxpQLwYnZmUw-0

Reply:
I just verified this by Christine, right now "hullerlog" should have it's own page and not be a tab.